zookeeper/server/init.sls

```diff
@@ -24,15 +24,15 @@ zookeeper-config-dir:
     - require:
       - cmd: move-zookeeper-dist-conf
 
-{%- if zk.process_control_system is defined %}
+{%- if zk.process_control_system %}
 
   {%- if zk.restart_on_change %}
 
 zookeeper-in-supervisord:
   cmd.run:
     - name: "{{ zk.pcs_restart_command }}"
     - require:
-      - pkg: {{ zk.process_control_system }}
+      - pkg: {{ zk.process_control_system_pkg }}
```

Comment:
I really never went this path myself, but looks like here we have some inconsistency.
There are no `pkg` states at all within `zookeeper-formula` scope. So for this to work you must include correct PCS provisioning state into a highstate run, otherwise you will always get an SLS compilation error.

I suggest to add a simple `pkg.installed` state before the `cmd.run`, or completely remove the requisite and rely on the fact that required tool should be installed somehow beforehand.

Reply:
Hi @vutny,

thanks for your feedback, right there is no ``pkg.installed`` in this formula for supervisor. From my prospective it would be better to install e.g supervisor in a dedicated formula. That for i will remove ``zk.process_control_system_pkg`` from my merge request.